README.md

```diff
@@ -18,7 +18,30 @@ This Digital Twin Definition Language (DTDL) Model Generator parses your DTDL js
 - *Namespace*: The namespace that will be injected into the generated model classes.
 - *JsonModelsDirectory*: The directory that contains the DTDL json files that we'll be parsing.
 - *OutputDirectory*: The directory that the generated model classes will be placed in.
-- *IncludeTemplateProject*: Whether or not to include the template project in the output directory.
+- *CopyrightHeader*: The header comment that contains copyright information.
+
+## Prerequisites
+The following are some Prerequisites/assumptions to be considered:
+
+### For Developers
+- .Net 6 must be installed, while building the model generator locally. Backward compatibility is not supported.
+- The system should have Git installed as the model generator uses MinVer package which requires Git.
+
+### For Users
+- The following classes have been copied at the location where model generator assembly is being executed:
+  - Extensions.cs
+  - ModelHelper.cs
+  - Relationship.cs
+  - RelationshipCollection.cs
+  - RelationshipEqualityComparer.cs
+  - SourceValueAttribute.cs
+  - TwinEqualityComparer.cs
+- The ModelGeneratorOptions property "CopyrightHeader" should have "// " prefix.
+
+## Limitations
+- The model generator do not support Telemetry fields and Component.
```

Comment:
1. I think what would be more digestible for developers is a table that shows types supported in DTDL and whether or not they're supported in the generator with a green checkmark or red X.
2. Array is not supported in DTDL V2, so let's leave that concept out.

Reply:
Done.